src/main/java/com/monumental/models/Monument.java

```diff
@@ -41,11 +42,8 @@
     @NotNull(groups = {New.class, Existing.class}, message = "Material can not be null")
     private String material;
 
-    @Column(name = "lat")
-    private Double lat;
-
-    @Column(name = "lon")
-    private Double lon;
+    @Column(name = "point", columnDefinition = "geometry")
+    private Point point;
```

Comment:
[Point point Point, point point Point point, point point. Point. Point, point.](https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo)

Reply:
Also this should probably be `Point coordinates` to be more descriptive